main-command/src/main/scala/sbt/Command.scala

```diff
@@ -187,6 +187,13 @@ object Command {
 
   def spacedC(name: String, c: Parser[Char]): Parser[String] =
     ((c & opOrIDSpaced(name)) ~ c.+) map { case (f, rem) => (f +: rem).mkString }
+
+  implicit class CommandWithName(cmd: Command) {
+    def commandName: Option[String] = cmd match {
```

Comment:
I wonder if this is a good name. Should we just call it `name`, or is it that a bad idea? Maybe `getName`? 🤔

Reply:
We cannot call it name because `SimpleCommand` already defines a `name` argument of different type: `String`.

Reply:
I know. So on `SimpleCommand` you get `String`, on Command you'll get an `Option[String]` because it might or might not have a name.

But again, it might be a bad idea. How about `nameOption`?

Reply:
`nameOption` sounds better imho